shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DMLStatement.g4

```diff
@@ -101,15 +101,27 @@ multipleTableNames
     ;
 
 select 
-    : unionClause
+    : selectSubquery forUpdateClause?
+    ;
+
+selectSubquery
+    : (queryBlock | selectUnionClause | parenthesisSelectSubquery) orderByClause? rowLimitingClause
+    ;
+
+selectUnionClause
+    : ((queryBlock | parenthesisSelectSubquery) orderByClause? rowLimitingClause) ((UNION ALL? | INTERSECT | MINUS) selectSubquery)+
```

Comment:
`rowLimitingClause` miss ?

Reply:
Why put `((UNION ALL? | INTERSECT | MINUS) selectSubquery)+` behind of the sentence?

Reply:
When we add `?` to `rowLimitingClause`, ANTLR gives `rule selectUnionClause contains an optional block with at least one alternative that can match an empty string` warning. So I didn't add it. Shall I add it?

Reply:
In `selectUnionClause`, for the first `selectSubquery` because of mutually left-recursive error, I specified it as `((queryBlock | parenthesisSelectSubquery) orderByClause? rowLimitingClause)` then I specified `((UNION ALL? | INTERSECT | MINUS) selectSubquery)+`

Reply:
> When we add `?` to `rowLimitingClause`, ANTLR gives `rule selectUnionClause contains an optional block with at least one alternative that can match an empty string` warning. So I didn't add it. Shall I add it?
As the definition of the 'rowLimitingClause' is below:
```
rowLimitingClause
    : (OFFSET offset (ROW | ROWS))? (FETCH (FIRST | NEXT) (rowcount | percent PERCENT)? (ROW | ROWS) (ONLY | WITH TIES))?
    ;
```
I think there is no need to add '?'.